src/flags.go

```diff
@@ -84,6 +84,9 @@ func intialiseFlags() {
 			"The format can be set explicitly via --in. See "+GithubRepositoryLink)
 	AssertSuccess(rootCmd.MarkFlagRequired("data-text"))
 
+	flags.BoolVarP(&CurrentConfig.NoDefaultHeaders, "no-default-headers", "n", false,
+		"Default headers (e.g. `Content-Type`) will not be passed to curl. Assumes --curl. Use \"-n -H 'Content-Type: FooBar'\" to override the default content type.")
```

Comment:
```suggestion
		"Default headers (e.g. "Content-Type") will not be passed to curl. Assumes --curl. Use \"-n -H 'Content-Type: FooBar'\" to override the default content type.")
```

It seems, that the backticks \` are used to add it to the usage string. We don't want that here